code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -349,6 +349,13 @@ GLOBAL_LIST_EMPTY(lifts)
 
 				playsound(collided_wall, 'sound/effects/meteorimpact.ogg', 100, TRUE)
 
+			if(ismineralturf(dest_turf))
+				var/turf/closed/mineral/M = dest_turf
+				for(var/mob/client_mob in SSspatial_grid.orthogonal_range_search(M, SPATIAL_GRID_CONTENTS_TYPE_CLIENTS, 8))
+					if(get_dist(dest_turf, client_mob) <= 8)
+						shake_camera(client_mob, 2, 3)
```

Comment:
Named parameters for 2 and 3

https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#prefer-named-arguments-when-the-meaning-is-not-obvious